src/panel/poi_panel.js

```diff
@@ -139,12 +149,20 @@ PoiPanel.prototype.setPoi = async function (poi, options = {}) {
 }
 
 PoiPanel.prototype.center = function() {
-  Telemetry.add(Telemetry.POI_GO)
+  if (this.poiSource && this.poiSource == 'pagesjaunes') {
```

Comment:
Following my last comment, now that i saw more code, a constant should be used to reference the poi sources 👍

Reply:
We're currently talking about using the API response to set the poiSource.